lib/Raven/Client.php

```diff
@@ -979,13 +672,43 @@ private function isEncodingCompressed()
         return 'gzip' === $this->config->getEncoding();
     }
 
-    private function autoloadRavenStacktrace()
```

Comment:
Why this has been removed? It's still needed.

Reply:
I cannot reproduce the bug anymore so I assumed it was somehow fixed as it's pretty old (PHP 5.2)... I tried testing with the code found in this issue: https://bugs.php.net/bug.php?id=42098.